prompt.py

```diff
@@ -791,50 +784,48 @@ def configure(self, args):
             c1 = get_arg(args, 1).lower()
             if c1 is not None:
                 if c1 == 'on' or c1 == '1':
-                    print("debug logging is now enabled")
+                    print("Debug logging is now enabled")
                     settings.set_loglevel(logging.DEBUG)
                 if c1 == 'off' or c1 == '0':
-                    print("debug logging is now disabled")
+                    print("Debug logging is now disabled")
                     settings.set_loglevel(logging.INFO)
 
             else:
-                print("cannot configure log.  Please specify on or off")
+                print("Cannot configure log. Please specify on|off")
 
         elif what == 'sc-events':
             c1 = get_arg(args, 1).lower()
             if c1 is not None:
                 if c1 == 'on' or c1 == '1':
-                    print("smart contract event logging is now enabled")
+                    print("Smart contract event logging is now enabled")
                     settings.set_log_smart_contract_events(True)
                 if c1 == 'off' or c1 == '0':
-                    print("smart contract event logging is now disabled")
+                    print("Smart contract event logging is now disabled")
                     settings.set_log_smart_contract_events(False)
 
             else:
-                print("cannot configure log.  Please specify on or off")
+                print("Cannot configure log. Please specify on|off")
 
         else:
-            print("cannot configure %s", what)
-            print("Try 'config sc-events on|off' or 'config debug on|off'")
+            print("Cannot configure %s try 'config sc-events on|off' or 'config debug on|off'", what)
 
     def parse_result(self, result):
         if len(result):
-            commandParts = [s for s in result.split()]
-            return commandParts[0], commandParts[1:]
+            command_parts = [s for s in result.split()]
+            return command_parts[0], command_parts[1:]
         return None, None
 
     def run(self):
-
         dbloop = task.LoopingCall(Blockchain.Default().PersistBlocks)
         dbloop.start(.1)
 
         Blockchain.Default().PersistBlocks()
 
         tokens = [(Token.Neo, 'NEO'), (Token.Default, ' cli. Type '),
-                  (Token.Command, "'help' "), (Token.Default, 'to get started')]
+                  (Token.Command, "\'help\' "), (Token.Default, "to get started")]
 
         print_tokens(tokens, self.token_style)
-        print("\n")
+        print('\n')
```

Comment:
why?

Reply:
It's the only `print("\n")` (1 out of 7), for the sake of consistency...